testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/ImpersonationTest.java

```diff
@@ -228,8 +241,9 @@ protected void testForbiddenImpersonation(String admin, String adminRealm) {
         Keycloak client = createAdminClient(adminRealm, establishClientId(adminRealm), admin);
         try {
             client.realms().realm("test").users().get(impersonatedUserId).impersonate();
+            Assert.fail("Expected ClientErrorException wasn't thrown.");
         } catch (ClientErrorException e) {
-            Assert.assertTrue(e.getMessage().indexOf("403 Forbidden") != -1);
+            Assert.assertTrue(e.getMessage().contains("403 Forbidden"));
```

Comment:
```suggestion
            Assert.assertThat(e.getMessage(), containsString("403 Forbidden"));
```
- to get better error message if that fails